forte/orbital-helpers/orbital_embedding.cc

```diff
@@ -469,6 +476,10 @@ std::shared_ptr<MOSpaceInfo> make_embedding(psi::SharedWavefunction ref_wfn,
         A_docc = options->get_int("NUM_A_DOCC");
         A_uocc = options->get_int("NUM_A_UOCC");
         outfile->Printf("\n  Number of A occupied/virtual MOs set to %d and %d\n", A_docc, A_uocc);
+    } else if (options->get_str("EMBEDDING_CUTOFF_METHOD") == "CORRELATED_BATH") {
+        print_h2("Orbital partition done according to threshold t1 and t2=t1/10000 to build a "

```

Comment:
What is t1? Seems no point to print t2?

Reply:
t1 is the embedding_threshold we set, and t2 is used to chop away some totally non-overlaping orbitals, this approximation will reduce the cost for ASET2. I will modify this printing.